src/zope/app/apidoc/utilities.py

```diff
@@ -325,10 +366,10 @@ def renderText(text, module=None, format=None, dedent=True):
 
     assert format in _format_dict.values()
 
-    text = dedentString(text)
-
-    if not isinstance(text, unicode):
+    if isinstance(text, bytes):
         text = text.decode('latin-1', 'replace')
```

Comment:
That's a tough one. What encoding are files and docstrings most likely to be in (still)?  latin-1 files are probably still predominant on Windows systems.